sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClientCodeSnippets.java

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.data.tables.implementation.models.TableServiceErrorException;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class TableClientCodeSnippets {
+
+    public static void methods() {
+
+        //create a tableServiceClient
+        TableServiceClient tableServiceClient = new TableServiceClientBuilder()
+            .connectionString("connectionString")
+            .buildClient();
+
+        //create TableClient
+        TableClient tableClient = new TableClientBuilder()
+            .connectionString("connectionString")
+            .tableName("OfficeSupplies")
+            .buildClient();
+
+
+        //create a table
+        try {
+            tableServiceClient.createTable("OfficeSupplies");
+        } catch (TableServiceErrorException e) {
+            //use azure core errors? based on
+            System.out.println("Create Table Unsuccessful. Error: " + e);
+        }
+
+        //delete  table
+        try {
+            tableServiceClient.deleteTable("OfficeSupplies");
+        } catch (TableServiceErrorException e) {
+            System.out.println("Delete Table Unsuccessful. Error: " + e);
+        }
+
+        //query tables
+        String selectString = "$selectString= TableName eq 'OfficeSupplies'";
```

Comment:
I think this is actually a `filter` string - just need to update the name (`String filterString`) and the prefix (`$filter=`)

Reply:
Also not sure if it's required, but all the example I see for the OData syntax seem to have no whitespace around the equals sign, so maybe go through and remove it in any of your OData filters and selects just to be consistent.